app/src/main/java/com/codefororlando/petadoption/persistence/mapper/AnimalMapper.kt

```diff
@@ -0,0 +1,33 @@
+package com.codefororlando.petadoption.persistence.mapper
+
+import com.codefororlando.petadoption.data.model.Animal
+import com.codefororlando.petadoption.persistence.model.AnimalEntity
+import javax.inject.Inject
+
+
+class AnimalMapper @Inject constructor(){
```

Comment:
This seems to be a utility class which means it should not be a class in kotlin. Instead of placeholder classes purely to provide extension functionality just write extension functions and get rid of the container class.

```kotlin
fun Animal.mapToAnimalEntity(): AnimalEntity
   ...
```